docs/getting-started/ios/screenshots.md

```diff
@@ -62,7 +62,7 @@ To jump-start your UI tests, you can use the UI Test recorder, which you can sta
 ## Setting Up _snapshot_
 
 1. Create a new UI Test target in your Xcode project (See the top part of [this article](https://krausefx.com/blog/run-xcode-7-ui-tests-from-the-command-line))
-1. Run `fastlane snapshot init` in your project folder
+1. Run (if you haven't already done so also run `fastlane init`) `fastlane snapshot init` in your project folder
```

Comment:
This is confusing a little bit, maybe we can do

```
1. If you haven't already done so run `fastlane init`
1. If you already have a fastlane setup, use `fastlane snapshot init` in your project folder
```

Reply:
I find

> If you haven't already done so, run `fastlane init`

easier to understand.